jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -606,23 +607,10 @@ final boolean goInDirection(EDirection direction, boolean force) {
 	}
 
 	/**
-	 * Forces the movable to go a step in the given direction (if it is not blocked).
-	 * 
-	 * @param direction
-	 *            direction to go
-	 */
-	final void forceGoInDirection(EDirection direction) {
```

Comment:
This method does something different than goInDirection(direction,true). It's true, the naming is bullshit here, as both are called `force`. However, I think that both functionalities are needed... but the names should definitely be changed. I'm quite sure that this is a game logic change.